src/index.html

```diff
@@ -0,0 +1,110 @@
+                                <input type="radio" name="options" autocomplete="off" checked> Tag
+                            </label>
+                            <label id="branch-option" class="btn btn-info">
+                                <input type="radio" name="options" autocomplete="off"> Branch
+                            </label>
+                        </div>
+
+                        <select class="custom-select d-block" id="select-option" required="" disabled>
+                            <option value="">Choose...</option>
+                        </select>
+                    </div>
+                    <div class="invalid-feedback" id="option-feedback"></div>
+                </div>
+            </div>
+        </form>
+
+        <hr class="mb-4">
+
+        <div class="row" id="repo-preview" style="display: none;">
+            <div class="col-md-6">
+                <h3 id="name">Bow</h3>
+                <div>
+                    <span class="text-light bg-dark text-center rounded-pill cloud-tag" id="owner"></span>
+                    <span class="text-light bg-dark text-center rounded-pill cloud-tag" id="tag-branch"></span>
+                </div>
+                <div class="row py-3">
+                    <div class="col-md-2">
+                        <img id="avatar" class="rounded-pill"
+                            src="https://avatars3.githubusercontent.com/u/44965417?v=4" alt="" width="64" height="64">
+                    </div>
+                    <div class="col-md-7 my-auto mycontent-right">
+                        <p class="text-secondary" id="repo-description"> Functional programming in Swift.</p>
+                    </div>
+                </div>
+            </div>
+
+            <div class="col-md-6">
+                <img src="https://raw.githubusercontent.com/bow-swift/bow-art/master/badges/nef-playgrounds-badge.svg"
+                    style="height:20px">
+                <div class="form-group purple-border h-100 mycontent-bottom textarea-container">
+                    <label for="badge-code">Code</label>
+                    <textarea class="form-control rounded-lg" id="badge-code" rows="9"
+                        style="font-size: 10pt"></textarea>
+                    <button id="copy" type="button" class="btn btn-light btn-outline-info btn-sm">
+                        <span class="fa fa-copy"></span> <span id="copy-value">Copy</span>
+                    </button>
+                </div>
+            </div>
+        </div>
+
+    </div>
+
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+        integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+        crossorigin="anonymous"></script>
```

Comment:
```suggestion
```

For the current functionality, it doesn't seem like this is needed, you can remove it to save some bits.

Reply:
Yep! I added it firstly for the selector we have, but you are right, in the end I am using the native `bootstrap` option.